_posts/2023-06-23-assembly-versions.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: Fun with assembly versions
+---
+
+Assembly loading remains a gift that keeps on giving. On a previous blog post I've focused on the assembly-resolving logic that tries to load an assembly which isn't already loaded but is deployed along with our application. This post focuses specifically on the type-resolving behavior when trying to retrieve the type from an assembly that is already loaded but with a different version in the provided fully qualified assembly name. Here's a rough diagram explaining the deployment:
+
+![](/assets/assembly-version-overview.png)
```

Comment:
What about using Mermaid?

Reply:
I was thinking about this but I don't think mermaid has a good graph option for this?

Reply:
the screenshot works just fine